modules/installation-about-custom-aws-vpcs.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * installing/
+
+[id="installation-about-custom-aws-vpcs_{context}"]
+= About using a custom VPC
+
+In {product-title} {product-version}, you can deploy a cluster into an existing VPC in Amazon Web Services (AWS). If you do, you must also use existing subnets within the VPC and routing rules.
+
+By deploying {product-title} into an existing AWS VPC, you might be able to avoid limit constraints in new accounts or more easily abide by the operational constraints that your company's guidelines set. This is a good option to use if you cannot obtain the infrastructure creation permissions that are required to create the VPC yourself.
+
+[id="installation-about-custom-aws-vpcs-requirements_{context}"]
+== Requirements for using your VPC
+
+The installation program will no longer create the following components:
+* Internet gateways
+* NAT gateways
+* Subnets
+* Route tables
+* VPCs
+* VPC DHCP options
+* VPC endpoints
+
+If you use a custom VPC, you must correctly configure it and its subnets for the installation program and the cluster to use. The installation program cannot subdivide network ranges for the cluster to use, set route tables for the subnets, or set VPC options like DHCP, so you must do so before you install the cluster.
+
+Your VPC must meet the following characteristics:
+
+* The VPC’s CIDR block must contain the `Networking.MachineCIDR` range, which is the IP address pool for cluster machines.
+* The VPC must not use the `kubernetes.io/cluster/.*: owned` tag.
```

Comment:
Do we enforce this?  I don't see a problem with allowing this.  For example, the user may want to install into pre-existing subnets in a VPC they are giving completely to the cluster because they want to set it up with an egress proxy.

Reply:
I'm not sure if we do - it was called out in the design doc. @abhinavdahiya, will you PTAL?